builtins/amp-img/amp-img.js

```diff
@@ -130,8 +131,9 @@ export class AmpImg extends BaseElement {
           this.element
         );
       }
-      this.propagateAttributes(
+      propagateAttributes(
```

Comment:
Unrelated to this change: I wonder how much we could save from compiling out these `BaseElement` helpers everywhere.

Reply:
After looking further, it looks like there's only two.